app/models/user.rb

```diff
@@ -67,7 +67,7 @@ class User < ApplicationRecord
     where(document_type: document_type, document_number: document_number)
   end
   scope :email_digest,   -> { where(email_digest: true) }
-  scope :active,         -> { where(erased_at: nil) }
+  scope :active,         -> { where(erased_at: nil).where.not(confirmed_at: nil) }
```

Comment:
Changing the `active` scope might have some side effects :thinking:. For instance, this scope is used while verifying users. What do you think about changing the `UserSegments.all_users` method instead, which will only affect newsletters?

Reply:
@javierm You are right about this, we don't know what might break. I will revert and do the change to all_users method